### PR TITLE
Add knex migrations

### PR DESCRIPTION
**Pull request by mtcrouse**

### CODE REVIEW

File: knex.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict';
```
Comment (MrJadaml) [inline knex.js]: 'use strict' is unnecessary inside of modules  strict

File: knexfile.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+'use strict';
```
Comment (MrJadaml) [inline knexfile.js]: 'use strict' is unnecessary inside of modules

File: knexfile.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+module.exports = {
+  development: {
+    client: 'pg',
+    connection: 'postgres://localhost/cadence_dev'
```
Comment (MrJadaml) [inline knexfile.js]: Missing trailing comma here and other objects in this file.

File: package.json [hunk 4]
```diff
@@ -7,7 +7,8 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "./node_modules/.bin/webpack -d",
     "lint": "./node_modules/.bin/eslint app/* server.js; exit 0",
-    "start": "nodemon server.js"
+    "start": "nodemon server.js",
+    "knex": "knex"
```
Comment (MrJadaml) [inline package.json]: this points at a global knex module. Instead have it point at the module in this app.
`./node_modules/.bin/knex`

**Review by MrJadaml — CHANGES_REQUESTED** [review]
Just a few small changes that popped up when I linted the code. As well as that local node module issue.

File: knexfile.js (hunk 3 above)
Comment (mtcrouse) [reply knexfile.js]: I've never used trailing commas before... I thought that caused a syntax error? But I guess not.

**Pull request merged**